gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/spring/ServiceConfiguration.java

```diff
@@ -102,6 +103,7 @@ public EventManager eventManager() {
     }
 
     @Bean
+    @Primary
```

Comment:
Don't know if we need to add a comment on it to avoid to see anybody trying to remove it... 🤷🏼

Reply:
I will add one ;)